proxygen_cli/test/command_instance_test.py

```diff
@@ -211,6 +211,46 @@ def test_instance_deploy_invalid_instance_name(patch_pathlib, patch_access_token
         "Invalid instance https://internal-dev.api.service.nhs.uk/mock-api-base-path"
         in result.output.strip())
 
+def test_instance_deploy_invalid_base_path(patch_pathlib, patch_access_token, patch_request):
+    env = "internal-dev"
+
+    runner = CliRunner()
+    with patch(
+        "proxygen_cli.lib.proxygen_api.access_token"
+    ) as _access_token, patch_request(404, {}), patch_pathlib(
+        "test-yaml-key: test-yaml-value"
+    ):
+        _access_token.return_value = "12345"
+        result = runner.invoke(
+            cmd_instance.deploy,
+            [env, "mock-api/base/path", "specification/mock-api-spec", "--no-confirm"],
+            obj={"api": "mock-api"},
+        )
+    assert (
+        "Multipart basepaths must include '_' instead of '/'"
+        in result.output.strip())
+
+
+def test_instance_deploy_valid_base_path(patch_pathlib, patch_access_token, patch_request):
```

Comment:
Worth calling out this is a valid multi part base path in the test name I think

Reply:
Have changed it to `test_instance_deploy_valid_multipart_base_path`